Documentation/ApiOverview/SymfonyExpressionLanguage/Index.rst

```diff
@@ -180,7 +181,7 @@ All further arguments need to be provided by TypoScript. The above example could
        page.10.value = Matched
    [GLOBAL]
 
-In case a simple string like a title is returned this can be further checked:
+In the case when a simple string like a title is returned, this can be further checked:
```

Comment:
```suggestion
If a simple string like a title is returned, this can be further checked:
```
Neither version is correct apparently:

https://dictionary.cambridge.org/grammar/british-grammar/in-case-of

Reply:
undone